inputfiles/addedTypes.jsonc

```diff
@@ -1126,6 +1126,17 @@
                         }
                     }
                 }
+            },
+            "HTMLStyleElement": {
+                "name": "HTMLStyleElement",
```

Comment:
This line is redundant, please remove it.

Reply:
Thank you, I updated the code to remove it